sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormField.java

```diff
@@ -38,29 +38,21 @@
      */
     private final FieldText valueText;
 
-    /*
-     * The 1 based page number.
-     */
-    private final Integer pageNumber;
-
     /**
      * Constructs a FormField object.
-     *
-     * @param confidence The confidence of the recognized field.
+     *  @param confidence The confidence of the recognized field.
```

Comment:
hmm, doesn't it has a checkstyle issue?

```suggestion
     * @param confidence The confidence of the recognized field.
```

Reply:
It doesn't apparently, maybe because it is Javadocs?